### DISCUSSION TITLE
End of Line

### DISCUSSION THREAD

**Discussion by kitswas** [rust-lang/rustfmt#5513] [Q&A] [answered]
Does rustfmt have a configuration option for the end-of-line-sequence(LF / CRLF/ CR)?

I did not find anything in [the documentation](https://rust-lang.github.io/rustfmt/?version=v1.5.1&search=line).

**Comment by calebcartwright** [comment, marked as answer]
Yes indeed - https://rust-lang.github.io/rustfmt/?version=v1.5.1&search=#newline_style

**Reply by Nadelio** [reply]
I can't find that config option anymore, has it been removed?

**Reply by ytmimi** [reply]
@Nadelio [newline_style](https://rust-lang.github.io/rustfmt/?version=v1.8.0&search=#newline_style) is still available.

**Reply by Nadelio** [reply]
Yea I found it when I looked it up directly instead of using the hyperlink OP provided.
